page_creation.rst

```diff
@@ -216,7 +216,9 @@ Make sure that ``LuckyController`` extends Symfony's base
 Now, use the handy ``render()`` function to render a template. Pass it a ``number``
 variable so you can use it in Twig::
 
+    <?php
```

Comment:
This is not needed

Reply:
Oh ok thanks @OskarStark  :) i noticed this